gregorio/PKGBUILD

```diff
@@ -0,0 +1,44 @@
+# Maintainer: Br Anthony VanBerkum <anthonyvbop AT gmail DOT com>
+# Contributor: Laércio de Sousa <lbsousajr AT gmail DOT com>
+# Contributor: David Gippner davidgippner at googlemail dot com
+pkgbase=gregorio
+pkgname=$pkgbase
+pkgver=2.4.2
+pkgrel=1
+pkgdesc="Command-line tool to typeset Gregorian chant"
+url=http://home.gna.org/gregorio
+arch=("i686" "x86_64")
+license=("GPL")
+depends=("texlive-core" "texlive-fontsextra" "texlive-bin" "texlive-formatsextra" "texlive-latexextra")
+conflicts=("gregorio-svn" "gregorio-git" "gregoriotex")
+provides=("gregorio-git")
```

Comment:
Fixed, thanks. I looked at some other examples, and I think they should both be `provides=("gregorio")`; it looks like it's generally used to make it clear that `foo` and `foo-git` provide the same package.